mnefun/_mnefun.py

```diff
@@ -3019,61 +3030,51 @@ def _spherical_conductor(info, subject, pos):
 
 
 def annotate_head_pos(raw, head_pos, rotation_limit=45, translation_limit=0.1,
-                      coil_dist_limit=0.005, t_step=0.05, t_window=0.05,
-                      prefix='  ', coil_bad_count_duration_limit=0.1):
-    u"""Annotate a raw instance based on bad head positions.
+                      fit_t=None, counts=None, prefix='  ',
+                      coil_bad_count_duration_limit=0.1):
+    u"""Annotate a raw instance based on coil counts and head positions.
 
     Parameters
     ----------
     raw : instance of Raw
         The raw instance.
-    head_pos : ndarray
-        The head positions.
+    head_pos : ndarray | None
+        The head positions. Can be None if movement compensation is off
+        to short-circuit the function.
     rotation_limit : float
         The rotational velocity limit in °/s.
         Can be infinite to skip rotation checks.
     translation_limit : float
         The translational velocity limit in m/s.
         Can be infinite to skip translation checks.
-    coil_dist_limit : float
-        The distance limit for coil distances (m).
-        Can be infinite to skip the use of coil distances.
-    t_step : float
-        The time step for coil calculations.
-    t_window : float
-        The time window for coil calculations.
+    fit_t : ndarray
+        Fit times.
+    counts : ndarray
+        Coil counts.
+    prefix : str
+        The prefix for printing.
     coil_bad_count_duration_limit : float
         The lower limit for bad coil counts to remove segments of data.
 
```

Comment:
Need doc string for `coil_t_window`, and default unit for `coil_bad_count_duration_limit`